css/style.css

```diff
@@ -1,7 +1,242 @@
-body{
-    margin: 0 1rem;
-}
+
+/* Extra small devices (phones, less than 768px) */
 .header{
-    background-color: #F72210;
-    color: #FFF;
-}
\ No newline at end of file
+  justify-content: start;
+}
+.main-header {
+  font-size: 1.3rem;
+}
+.navbar-toggler-icon{
+  background-image: url("../images/hamburger_button.png");
+}
+.header-text a{
+  font-size: .7rem;
+}
+.bg-color01{
+  background-color: #C2D4D7;
+}
+.icon-search{
+  width: 2rem;
+  height: auto;
+}
+.logo{
+  width: 2rem;
+  height: 2rem;
+}
+.icon-slideshow{
+  width: 4rem;
+  height: 3rem;
+  background-image: url(https://g.newsweek.com/themes/newsweek/images/icon-slideshows-nw.png);
+  background-size: contain;
+  background-repeat: no-repeat;
+  text-indent: .4rem;
+  line-height: 2.5rem;
+}
+.cover-back, .cover-front{
+  width: 170px;
+  height: 228px;
+}
+.cover-front{
+  left: 4rem;
+  top: 2rem;
+}
+
+.subscribe, .newsletter-part{
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
```

Comment:
`SUGGESTION`

Consider using a bootstrap class instead. _Note that the project is about using bootstrap._

Reply:
done